pgx/pgx.go

```diff
@@ -80,3 +87,82 @@ func (b *pgxBackend) QueryStructRow(result interface{}, query string, args ...in
 func (b *pgxBackend) QueryWriteCSV(w io.Writer, options onedb.CSVOptions, query string, args ...interface{}) error {
 	return onedb.QueryWriteCSV(w, options, b, query, args...)
 }
+
+type pgxTx struct {
+	tx *pgx.Tx
+	Txer
+}
+
+type Txer interface {
+	Commit() error
+	Conn() *pgx.Conn
+	Rollback() error
+	Status() int8
+	PGXQuerier
+}
+
+type PGXQuerier interface {
```

Comment:
The idea behind making it public is so that functions that use `onedb/pgx` don't have to decide whether they take a `PGXer` or a `TXer`. They can take a `PGXQuerier`, which both `PGXer` and `TXer` will satisfy.

For example:
```
func insertProvidersIntoDB(providers []contractProvider, sixDegRosterID int32, claimsDB pgx.PGXQuerier) error {
	columnNames := []string{"sixdegrosterid", "name", "npi", "taxid"}
	interfaces := contractProvidersToInterfaces(providers, sixDegRosterID)

	if copyCount, err := claimsDB.CopyFrom(pgx.Identifier{"contractprovider"}, columnNames, pgx.CopyFromRows(interfaces)); err != nil {
		return fmt.Errorf("error saving duplicates (count %v): %v", copyCount, err)
	}

	return nil
}
```

doesn't need to know whether it's been fed a `PGXer` or a `TXer`.

Reply:
Fine, then at least add a comment so VSCode doesn't complain. Approving as is, but please include this with your next PR with the tests